tensorboard/plugins/image/tf_image_dashboard/tf-image-loader.html

```diff
@@ -34,34 +34,47 @@
 -->
 <dom-module id="tf-image-loader">
   <template>
-    <tf-card-heading tag="[[tag]]" run="[[run]]" color="[[_runColor]]">
+    <tf-card-heading
+      tag="[[tag]]"
+      run="[[run]]"
+      display-name="[[tagMetadata.displayName]]"
+      description="[[tagMetadata.description]]"
+      color="[[_runColor]]"
+    >
+      <template is="dom-if" if="[[_hasMultipleSamples]]">
+        <div>sample: [[_sampleText]] of [[ofSamples]]</div>
+      </template>
       <template is="dom-if" if="[[_hasAtLeastOneStep]]">
-        step
-        <span style="font-weight: bold">[[_stepValue]]</span>
-        <template is="dom-if" if="[[_currentWallTime]]">
-          ([[_currentWallTime]])
-        </template>
-        <paper-spinner-lite active hidden$=[[!_isImageLoading]]>
-        </paper-spinner-lite>
+        <div>
+          step
+          <span style="font-weight: bold">[[_stepValue]]</span>
+          <template is="dom-if" if="[[_currentWallTime]]">
+            ([[_currentWallTime]])
+          </template>
+          <paper-spinner-lite active hidden$=[[!_isImageLoading]]>
+          </paper-spinner-lite>
+        </div>
       </template>
       <template is="dom-if" if="[[_hasMultipleSteps]]">
-        <paper-slider
-          id="steps"
-          immediate-value="{{_stepIndex}}"
-          max="[[_maxStepIndex]]"
-          max-markers="[[_maxStepIndex]]"
-          snaps
-          step="1"
-          value="{{_stepIndex}}"
-        ></paper-slider>
+        <div>
+          <paper-slider
+            id="steps"
+            immediate-value="{{_stepIndex}}"
+            max="[[_maxStepIndex]]"
+            max-markers="[[_maxStepIndex]]"
+            snaps
+            step="1"
+            value="{{_stepIndex}}"
+          ></paper-slider>
+        </div>
       </template>
     </tf-card-heading>
     <div id="main-image-container"></div>
 
     <style>
       :host {
         display: block;
-        width: 330px;
+        width: 350px;
```

Comment:
Note for posterity: I made this change because when summary descriptions are included, the line with the step and current wall time is just barely short enough to not wrap, but the `paper-spinner-lite`, when visible, causes it to wrap. Thus, dragging the step slider causes the image to flicker up and down, which is both extremely annoying and makes visually comparing adjacent steps much more difficult. (This surfaces only now because the width of the container is shrunk slightly by the addition of the "info" icon for the summary description.)